### PR TITLE
add Scala 2.13 support

### PR DESCRIPTION
**Pull request by xuwei-k**

### CODE REVIEW

**Comment by xuwei-k** [comment]
travis-ci build does not start 🤔 ???

File: .travis.yml [hunk 1]
```diff
@@ -4,10 +4,14 @@ script:
   - sbt "++${TRAVIS_SCALA_VERSION}" "^^${SBT_VERSION}" test scripted
```
Comment (jeffmay) [inline .travis.yml]: I can't comment on the jdk line, but maybe you should also test `openjdk8` and `openjdk11`?

**Comment by jeffmay** [comment]
I was having trouble using this with `crossScalaVersions` in my project.  It only seemed to want to `generateContrabands` for my Scala 2.13 target sourceManaged directory.

File: .travis.yml [hunk 2]
```diff
@@ -4,10 +4,14 @@ script:
...
 matrix:
   include:
-  - env: SBT_VERSION="0.13.17"
+  - env: SBT_VERSION="0.13.18"
     scala: 2.10.7
   - env: SBT_VERSION="1.2.8"
     scala: 2.12.8
+  - scala: 2.13.0
+    script: "++${TRAVIS_SCALA_VERSION}" library/test
```
Comment (eed3si9n) [inline .travis.yml]:
```suggestion
    script: "sbt \"++${TRAVIS_SCALA_VERSION}\" library/test"
```

File: .travis.yml [hunk 3]
```diff
@@ -4,10 +4,14 @@ script:
...
+  - scala: 2.11.12
+    script: "++${TRAVIS_SCALA_VERSION}" library/test
```
Comment (eed3si9n) [inline .travis.yml]:
```suggestion
    script: "sbt \"++${TRAVIS_SCALA_VERSION}\" library/test"
```

**Comment by eed3si9n** [comment]
When Travis CI doesn't start http://www.yamllint.com/ is your friend.

**Pull request merged**